src/validations/metadata-schema.ts

```diff
@@ -18,4 +18,35 @@ export const metadata = conditionalValidation({
     const errors = validator.errors?.map(($) => '' + $.message) || []
     return validationFailed(`The metadata for this entity type (${deployment.entity.type}) is not valid.`, ...errors)
   }
-})
+}
+
+type ADR = {
+  number: number
+  timestamp: number
+}
+
+const ADRMetadataVersionTimelines: Record<EntityType, ADR[]> = {
```

Comment:
can you add a comment here explaining this map?

Reply:
yup